.script/tests/KqlvalidationsTests/CustomTables/GCP_DNSV2_CL.json

```diff
@@ -0,0 +1,185 @@
+            "Type": "String"
+        },
+        {
+            "Name": "payload_authorizationInfo_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_resourceName_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_request__type_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_request_project_s",
+            "Type": "String"
+        },
+        {
+            "Name": "resource_labels_policy_name_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_request_managedZone_s",
+            "Type": "String"
+        },
+        {
+            "Name": "log_name_s",
+            "Type": "String"
+        },
+        {
+            "Name": "insert_id_s",
+            "Type": "String"
+        },
+        {
+            "Name": "severity_s",
+            "Type": "String"
+        },
+        {
+            "Name": "timestamp_t",
+            "Type": "DateTime"
+        },
+        {
+            "Name": "resource_type_s",
+            "Type": "String"
+        },
+        {
+            "Name": "resource_labels_project_id_s",
+            "Type": "String"
+        },
+        {
+            "Name": "resource_labels_target_type_s",
+            "Type": "String"
+        },
+        {
+            "Name": "resource_labels_source_type_s",
+            "Type": "String"
+        },
+        {
+            "Name": "resource_labels_target_name_s",
+            "Type": "String"
+        },
+        {
+            "Name": "resource_labels_location_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_vmProjectId_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_protocol_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_queryType_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_rdata_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_vmInstanceId_d",
+            "Type": "Double"
+        },
+        {
+            "Name": "payload_vmInstanceIdString_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_vmInstanceName_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_responseCode_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_authAnswer_b",
+            "Type": "Boolean"
+        },
+        {
+            "Name": "payload_queryName_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_vmZoneName_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_sourceNetwork_s",
+            "Type": "String"
+        },
+        {
+            "Name": "payload_serverLatency_d",
+            "Type": "Double"
+        },
+        {
+            "Name": "payload_sourceIP_s",

```

Comment:
For all fields naming convention kindly use pascal casing ,

Its a v2 Table , why , you are using  _s , _t as suffix with fields name